src/MetricsUploader/orbit_log_event.proto

```diff
@@ -227,4 +227,52 @@ message OrbitCaptureData {
 
   // Total number of callstack samples marked as unwinding error in the capture.
   int64 number_of_unwinding_errors = 28;
+
+  // State of the target process after the capture.
+  enum TargetProcessStateAfterCapture { 
+    TARGET_PROCESS_STATE_AFTER_CAPTURE_UNKNOWN = 0;
+    TARGET_PROCESS_STATE_AFTER_CAPTURE_INTERNAL_ERROR = 1;
+    TARGET_PROCESS_STATE_AFTER_CAPTURE_RUNNING = 2;
+    TARGET_PROCESS_STATE_AFTER_CAPTURE_ENDED = 3;
+    TARGET_PROCESS_STATE_AFTER_CAPTURE_CRASHED = 4;
+  }
+  TargetProcessStateAfterCapture target_process_state_after_capture = 29;
+
+  enum TargetProcessTerminationSignal {
+    TARGET_PROCESS_TERMINATION_SIGNAL_UNKNOWN = 0;
+    TARGET_PROCESS_TERMINATION_SIGNAL_SIGHUP = 1;
+    TARGET_PROCESS_TERMINATION_SIGNAL_SIGINT = 2;
+    TARGET_PROCESS_TERMINATION_SIGNAL_SIGQUIT = 3;
+    TARGET_PROCESS_TERMINATION_SIGNAL_SIGILL = 4;
+    TARGET_PROCESS_TERMINATION_SIGNAL_SIGTRAP = 5;
+    TARGET_PROCESS_TERMINATION_SIGNAL_SIGABRT = 6;
+    TARGET_PROCESS_TERMINATION_SIGNAL_SIGBUS = 7;
+    TARGET_PROCESS_TERMINATION_SIGNAL_SIGFPE = 8;
+    TARGET_PROCESS_TERMINATION_SIGNAL_SIGKILL = 9;
+    TARGET_PROCESS_TERMINATION_SIGNAL_SIGUSR1 = 10;
+    TARGET_PROCESS_TERMINATION_SIGNAL_SIGSEGV = 11;
+    TARGET_PROCESS_TERMINATION_SIGNAL_SIGUSR2 = 12;
+    TARGET_PROCESS_TERMINATION_SIGNAL_SIGPIPE = 13;
+    TARGET_PROCESS_TERMINATION_SIGNAL_SIGALRM = 14;
+    TARGET_PROCESS_TERMINATION_SIGNAL_SIGTERM = 15;
+    TARGET_PROCESS_TERMINATION_SIGNAL_SIGSTKFLT = 16;
+    TARGET_PROCESS_TERMINATION_SIGNAL_SIGCHLD = 17;
+    TARGET_PROCESS_TERMINATION_SIGNAL_SIGCONT = 18;
+    TARGET_PROCESS_TERMINATION_SIGNAL_SIGSTOP = 19;
+    TARGET_PROCESS_TERMINATION_SIGNAL_SIGSTP = 20;
+    TARGET_PROCESS_TERMINATION_SIGNAL_SIGTTIN = 21;
+    TARGET_PROCESS_TERMINATION_SIGNAL_SIGTTOU = 22;
+    TARGET_PROCESS_TERMINATION_SIGNAL_SIGURG = 23;
+    TARGET_PROCESS_TERMINATION_SIGNAL_SIGXCPU = 24;
+    TARGET_PROCESS_TERMINATION_SIGNAL_SIGXFSZ = 25;
+    TARGET_PROCESS_TERMINATION_SIGNAL_SIGVTALRM = 26;
+    TARGET_PROCESS_TERMINATION_SIGNAL_SIGPROF = 27;
+    TARGET_PROCESS_TERMINATION_SIGNAL_SIGWINCH = 28;
+    TARGET_PROCESS_TERMINATION_SIGNAL_SIGIO = 29;
+    TARGET_PROCESS_TERMINATION_SIGNAL_SIGPWR = 30;
+    TARGET_PROCESS_TERMINATION_SIGNAL_SIGSYS = 31;
+    TARGET_PROCESS_TERMINATION_SIGNAL_UNSPECIFIED = 32;
+    TARGET_PROCESS_TERMINATION_SIGNAL_INTERNAL_ERROR = 33;
+  };
+  TargetProcessTerminationSignal target_process_termination_signal = 30;
```

Comment:
This needs a change in google3 first. Will talk about this offline.

Reply:
Right, thnaks! Proto changes are in.